### DISCUSSION TITLE
Consistent Style - Exit button behavior.

### DISCUSSION THREAD

**Discussion by MichaelKelly9** [MichaelKelly9/Tafe24S1T104_Ext#7] [General]
Should the Exit button in the individual apps exit the program entirely? OR drop back to the Main Menu? what do you think @ZIHAOJIA

**Comment by ZIHAOJIA** [comment]
thanks for your remind. I will check the exit button tomorrow and fix it on F2 branc. i think it should drop back to the Main Menu.

**Reply by MichaelKelly9** [reply]
Agreed, but I may need your help with the c# code for the exit button. I only know how to get it to exit entirely. If you could let me know the correct code for dropping back to the main menu, I will update my feature branch with it before the merge.

**Reply by ZIHAOJIA** [reply]
created a exit_click event, then the code is 
	private void ExitButton_Click(object sender, RoutedEventArgs e)
	{
		Frame.Navigate(typeof(MainMenu));

	}

**Reply by ZIHAOJIA** [reply]
you should also created a click event on mainmenu button of currecnyConvertButton_Click, to navigate to your currecnyConvert page

**Reply by MichaelKelly9** [reply]
Thanks @ZIHAOJIA I think we can update the main menu button click events after we have completed our feature branches and merged them back into Dev. That way we have all the new pages etc in the same branch. 
I may ask for your help with the coding of those main menu buttons when we get to that point, as I haven't done C# code before this subject.

**Reply by ZIHAOJIA** [reply]
Hi Michael, i suggest you first modify your mai menu button before you merge your F1 to you dev,i will show you how to code through email

**Comment by ZIHAOJIA** [comment]
I upgrade the F2 branch already and please check it again. i fixed the exit button to banck to mainpage, and heading size color

**Reply by MichaelKelly9** [reply]
@ZIHAOJIA If you have already finished your feature branch mortgage calc, feel free to go ahead with Section C - part d. and merge your feature into the Dev branch and push to Github etc. I will get my branch finished soon and merge into Dev after you have merged yours.

I think once they are merged into the Dev we can then double check styles etc are consistent.

**Reply by ZIHAOJIA** [reply]
thanks for your remind.I will now commit my F2 to the dev and push it to the remote repo.
